### ISSUE TITLE
link tag with icons not aligning correctly

### ISSUE THREAD

**Issue by Waguilar33** [creativetimofficial/material-dashboard-pro#57]
Adding icons inside links doesn't render the icon vertically in the middle.
`<a href="#"><i class="material-icons">vpn_key</i> Forgot password?</a>`
![image](https://user-images.githubusercontent.com/21320293/28028319-60851a76-6559-11e7-847b-9ba3a93f3954.png)

**Comment by alexandru-paduraru** [comment]
@Waguilar33 can you please give us more details about the place that you used the links with icons? Just added your example in a card and everything was working fine. Please check this:

<img width="667" alt="screen shot 2017-07-11 at 15 18 51" src="https://user-images.githubusercontent.com/2517758/28068004-5f4a705e-664c-11e7-8350-038da3e61849.png">

Best,
Alex

**Comment by Waguilar33** [comment]
I was using your login page example and i needed the remember me? checkbox as well as the forgot password link. So I went ahead a tried to use the
```
<p class="category text-center">
    Or Be Classical (I replaced this with the link tag)
    <a href="#"><i class="material-icons">vpn_key</i> Forgot password?</a>
</p>
```

and the error did persist...
![image](https://user-images.githubusercontent.com/21320293/28075861-fb57b2f0-6619-11e7-8a6d-ccc0c2e2991b.png)

However, I went ahead and moved the tag below the go [GO] button and the link does align correctly
![image](https://user-images.githubusercontent.com/21320293/28075928-2eb3fd20-661a-11e7-94e9-becee30b98f6.png)

So probably the category text-center class is the one with the problem.

Thanks in advance for the help and the reply
